lib/articles.ts

```
@@ -1,117 +1,75 @@
 import { promises as fs } from "fs";
 import { compileMDX } from "next-mdx-remote/rsc";
 import path from "path";
-import z from "zod";
 
-import { categoryNames } from "@/lib/constants/categories";
+import {
+  type Error,
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)